src/dev/flang/tools/docs/Html.java

```diff
@@ -96,7 +96,7 @@ public Html(DocsOptions config, Map<AbstractFeature, Map<Kind,TreeSet<AbstractFe
    */
   private String inherited(AbstractFeature af)
   {
-    if (af.inherits().isEmpty())
+    if (af.inherits().isEmpty() || (Util.Kind.classify(af) == Util.Kind.Other)) // don't show inheritance for function features
```

Comment:
`Other` seems to include `Choice`, which should have the inheritance shown. 
I think it would be better to use `AbstractFeature.Kind` or `IR.FeatureKind` and not have a new `Util.Kind` which will run out out sync with the `fz` command.

Reply:
requires #3810

Reply:
I do not see how your reply is related to my comment.

Reply:
I will use the solution from #3810 here once it is merged. It was partly a reminder to myself why I have not resolved this yet.